apps/site/src/components/pages/user/overview-card.tsx

```diff
@@ -19,6 +19,29 @@ type OverviewCardProps = {
   hideImage?: boolean;
 };
 
+const TypeContent: FunctionComponent<{
+  type: OverviewCardProps["type"];
+}> = ({ type }) => {
+  if (type === "link-entity-type") {
+    return (
+      <>
+        Entity Type
+        <LinkIcon
+          sx={{
+            marginLeft: "0.25rem",
+            width: "1rem",
+            height: "1rem",
+          }}
+        />
```

Comment:
I wonder if we want to put the 🔗 to the left instead. (no strong opinion)
We could also (and/or?) use it as an image.

Reply:
(haven't pushed code yet)
![image](https://user-images.githubusercontent.com/6988668/222419871-d3a6808b-70ef-4ed5-a94a-bf1e5348b9ce.png)

Looks fine, thoughts?

Reply:
This is better, thanks @thehabbos007! 
I agree with Ciaran that we should keep the image, so that change sufficient.

Reply:
9852976a96f546f94bea4309f1bd38736a22ad39